src/ui/wxWidgets/addeditpropsheet.cpp

```diff
@@ -238,30 +238,30 @@ void AddEditPropSheet::Init()
 ////@end AddEditPropSheet member initialisation
 }
 
-class PolicyValidator : public MultiCheckboxValidator
-{
-public:
-  PolicyValidator(int rbID, int ids[], size_t num,
-      const wxString& msg, const wxString& title)
-    : MultiCheckboxValidator(ids, num, msg, title), m_rbID(rbID) {}
-  PolicyValidator(const PolicyValidator &other)
-    : MultiCheckboxValidator(other), m_rbID(other.m_rbID) {}
-  ~PolicyValidator() {}
-
-  wxObject* Clone() const {return new PolicyValidator(m_rbID, m_ids, m_count, m_msg, m_title);}
-  bool Validate(wxWindow* parent) {
-    wxWindow* win = GetWindow()->FindWindow(m_rbID);
-    if (win && win->IsEnabled()) {
-      wxRadioButton* rb = wxDynamicCast(win, wxRadioButton);
-      if (rb && rb->GetValue()) {
-  return true;
-      }
-    }
-    return MultiCheckboxValidator::Validate(parent);
-  }
-private:
-  int m_rbID;
-};
+// class PolicyValidator : public MultiCheckboxValidator
+// {
+// public:
+//   PolicyValidator(int rbID, int ids[], size_t num,
+//       const wxString& msg, const wxString& title)
+//     : MultiCheckboxValidator(ids, num, msg, title), m_rbID(rbID) {}
+//   PolicyValidator(const PolicyValidator &other)
+//     : MultiCheckboxValidator(other), m_rbID(other.m_rbID) {}
+//   ~PolicyValidator() {}
+
+//   wxObject* Clone() const {return new PolicyValidator(m_rbID, m_ids, m_count, m_msg, m_title);}
+//   bool Validate(wxWindow* parent) {
+//     wxWindow* win = GetWindow()->FindWindow(m_rbID);
+//     if (win && win->IsEnabled()) {
+//       wxRadioButton* rb = wxDynamicCast(win, wxRadioButton);
+//       if (rb && rb->GetValue()) {
+//   return true;
+//       }
+//     }
+//     return MultiCheckboxValidator::Validate(parent);
+//   }
+// private:
+//   int m_rbID;
+// };
```

Comment:
Why not just remove this code?

Reply:
I thought this might reflect an idea that will be followed up by somebody at a later time.

Reply:
Personally I think commented-out chunks are a code smell, but again this is a minor issue.

Reply:
@ronys I would also tend to remove this code if nobody has it on its radar.

Reply:
It's elegant, in its way, so I'm leaving it for now.
Note that if we remove it, we should also remove its base class in wxutil.cpp/h